phoenix-core/src/main/java/org/apache/phoenix/hbase/index/write/IndexWriter.java

```diff
@@ -131,25 +131,6 @@ public IndexWriter(IndexCommitter committer, IndexFailurePolicy policy,
     this.writer = committer;
     this.failurePolicy = policy;
   }
-  
-  /**
-   * Write the mutations to their respective table.
-   * <p>
-   * This method is blocking and could potentially cause the writer to block for a long time as we
-   * write the index updates. When we return depends on the specified {@link IndexCommitter}.
-   * <p>
-   * If update fails, we pass along the failure to the installed {@link IndexFailurePolicy}, which
-   * then decides how to handle the failure.
-   * @param indexUpdates Updates to write
- * @param clientVersion version of the client
- * @throws IOException 
-   */
-    public void writeAndHandleFailure(Collection<Pair<Mutation, byte[]>> indexUpdates,
```

Comment:
What about indexes using the old design? Shouldn't they get disabled / use the old failure policies until they're upgraded?

Reply:
This method was only used by the new design. So, I removed it as it was not necessary anymore.